### ISSUE TITLE
Self-resetting password entries & false history entries since 6th of June

### ISSUE THREAD

**Issue by konsumentenschutz** [padloc/padloc#752]
_Preface: We use the cloud-version of Padloc in our team (paid version), not the self-deployed one._

**TL;DR:** Passwords are resetting themselves to earlier values, also >100 history entries appeared by themselves.

### Self-resetting passwords and missing history

It has come to our attention this Monday that at least one password entry has self-reset itself to an earlier value. On at least 2 occasions we've had this same vault item self-resetting its password to the old value.

On Friday we have changed a password which is in a shared (organisation) vault. The password was saved and used on Friday successfully. Upon returning to work on Monday, suddenly the password is no longer working - it has reset itself back to the earlier value (before Friday). We thought that "maybe it's just a fluke", so we've set a new password on Monday and have again entered it into Padloc. **Yesterday (Tuesday) it has again reset itself back to the value before Friday!**
To add insult to injury, the history of the Padloc entry insists it wasn't changed before, but only on Tuesday, 11.06.24 at 15:00:27 (which was not the time we changed the password for the first or the second time). So basically it's gaslighting us as well as self-resetting the password.

### Inexplainable history entries

This of course made me curious (and nervous), so I've started to look at the history entries in our Padloc vault items. As it turns out there are randomly-seeming history entries for almost all passwords, ranging from 06.06.24 to 11.06.24. However, these non-changes appear to have been made within seconds - and not by anyone of our team. Upon further investigation I've found these kinds of inexplainable history entries in my personal vault items as well.

An incomplete list of dates/times of these weird history entries:

**Personal vault items:**
> **07.06.24** 09:56:31
> **07.06.24** 09:56:32
> **07.06.24** 09:56:33
> **07.06.24** 09:56:35
> **07.06.24** 09:56:36
> **07.06.24** 09:56:37
> **07.06.24** 09:58:38
> **07.06.24** 09:56:39
> **07.06.24** 10:16:37

**Team vault items:**
> **06.06.24** 08:39:17
> **06.06.24** 16:40:05
> **06.06.24** 16:40:06
> **07.06.24** 09:56:31
> **10.06.24** 13:15:29
> **10.06.24** 13:15:55
> **11.06.24** 09:53:44
> **11.06.24** 12:00:08
> **11.06.24** 15:00:27
> **11.06.24** 15:00:28

_Please note that there are multiple entries for the large majority of these times (e.g. 4 history entries for the same second). I could not find any entries of this nature before the 6th of June._
**None of these history entries were caused by me or a member of our team.**

I've decided to post this here so the devs can hopefully fix it, or others will at least see it and be able to check whether the above-mentioned issues persist for them as well.

As for us, trust into the core functionality of Padloc is no longer given and hence we will be moving on immediately.

**Comment by konsumentenschutz** [comment]
In the meantime I have found out about another password that has been reset (on 11.06.24, 15:00:27 according to history) to an earlier value by Padloc itself. It sure is annoying to have a password manager that **replaces recently changed passwords with old values** and then lies about it by changing the history of the entry.

Additionally I have changed a different password on Tueday as a test, to find out whether it will get changed back again. As I knew what could be happening I made screenshots of the entry on Tuesday (image 1). My suspicions came true, hence I made another screenshot today (Thursday, image 2).

Image 1 shows the entry on Tuesday (had to censor parts as I do not want to share PII). I changed the PIN to a long (fictional) password via Padloc and saved the entry (as seen in Image 1). 

![padlocapp-anwendercode_kopierer](https://github.com/padloc/padloc/assets/153183527/41112386-6979-470c-a5be-198dcf2d8461)

Image 2 shows the same entry today (Thursday) (again with censorship due to PII*). As you can see, Padloc changed the password back to the shirt four digits PIN. While at it, it also rewrote its history - there is no more entry for the 06.06.24 nor for the 11.06.24 as there was in image 1. They were erased without a trace. Instead there is a new entry for 12.06.24 (yesterday) at 15:21:22 which wasn't made by us.

![padlocapp-anwendercode_kopierer-reset](https://github.com/padloc/padloc/assets/153183527/880fbd4f-00bb-4122-a66e-6a1ef70f06a9)

*The resolution and dimensions are different due to me using a different device today.

I would recommend everyone using Padloc to check whether this issue applies to them as well.
All in all I sincerely hope that I'm wrong but it does seem an awful lot like the problem persists for multiple entries, if not all of them.

**Comment by giftedl** [comment]
The officially hosted Padloc instance is unmaintained, and its recommended to move to your own self-hosted instance instead. If you really think this might make a difference, there is a way to contact support (this most certainly is an issue with the instance itself, not the software) which could help, but I doubt support is even being looked at. Padloc is great software, but you might just need to move to self-hosting it or looking at another alternative.

**Comment by konsumentenschutz** [comment]
Yes, we've come to the conclusion it isn't being maintained (anymore) as well. The github issues seemingly keep piling up. If it isn't an error in the software, that's a start I guess - however it's not very comforting to us, as it's still not being fixed. Support has been contacted multiple times to no avail. Self-hosting is not an option due to none of our team-members having the skillset required to fix errors etc. Therefore we've cut our losses (yearly plan) and quickly migrated to a better maintained alternative.
